python/pyspark/pandas/internal.py

```diff
@@ -25,7 +25,13 @@
 import pandas as pd
 from pandas.api.types import CategoricalDtype  # noqa: F401
 from pyspark._globals import _NoValue, _NoValueType
-from pyspark.sql import functions as F, Column, DataFrame as SparkDataFrame, Window
+from pyspark.sql import (
+    functions as F,
+    Column,
+    DataFrame as LegacyDataFrame,
+    Window,
+    SparkSession as PySparkSession,
+)
```

Comment:
Should use only either `LegacyXxx` or `PySparkXxx`?

Reply:
Let me consolidate them into `PySparkxxx`